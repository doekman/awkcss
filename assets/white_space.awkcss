BEGIN {
	white_space(pre);
	width(25);
}

/^#/ {
	white_space(pre_wrap);
}
