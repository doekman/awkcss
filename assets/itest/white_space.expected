#123456789012345678901234
# Testing the wrap rap is
 not crap rap            
int void(argv main, argc 
    return argc/int;     
}                        
